### PR TITLE
Logging options flags

### PR DESCRIPTION
**Pull request by aryszka**

### CODE REVIEW

File: cmd/skipper/main.go [hunk 1]
```diff
@@ -103,6 +109,9 @@ func init() {
 	flag.StringVar(&metricsPrefix, "metrics-prefix", "", metricsPrefixUsage)
 	flag.BoolVar(&debugGcMetrics, "debug-gc-metrics", false, debugGcMetricsUsage)
 	flag.BoolVar(&runtimeMetrics, "runtime-metrics", true, runtimeMetricsUsage)
+	flag.StringVar(&applicationLog, "application-log", "", applicationLogUsage)
+	flag.StringVar(&applicationLogPrefix, "application-log-prefix", "", applicationLogPrefixUsage)
```
Comment (lmineiro) [inline cmd/skipper/main.go]: I thought we wanted some other default here. `APP`? `APPLICATION`? `APPLICATION_LOG`?

File: logging/log.go [hunk 2]
```diff
@@ -62,5 +56,5 @@ func Init(o Options) {
 	}
 }
 
-func ApplicationLog() *logrus.Logger { return appLog }
+func ApplicationLog() *logrus.Logger { return logrus.StandardLogger() }
```
Comment (lmineiro) [inline logging/log.go]: This one is, basically, available through `logrus.XXX` or, with the alias, `log.XXX`. Do you think it's worth it to export such function?

File: cmd/skipper/main.go (hunk 1 above)
Comment (aryszka) [inline cmd/skipper/main.go]: yup, we'll have it in our own appliance. notice that the default for the access log here is that it is turned off. The reason is that when someone just starts the app in dev mode, may not want to be flooded with access log. And then the prefix is not required. Is this a false logic?

File: logging/log.go (hunk 2 above)
Comment (aryszka) [inline logging/log.go]: true. actually, neither the applog or the accesslog is worth exporting.

**Pull request closed without merging**